os/services/lwm2m/lwm2m-engine.h

```diff
@@ -114,6 +116,14 @@ void lwm2m_notify_object_observers(lwm2m_object_instance_t *obj,
 
 void lwm2m_engine_set_opaque_callback(lwm2m_context_t *ctx, lwm2m_write_opaque_callback cb);
 
+#if LWM2M_Q_MODE_ENABLED
+uint8_t lwm2m_engine_is_waked_up_by_notification();
```

Comment:
For API functions, I'd really like to see some meaningful doxygen documentation. And yes I know the state of documentation in this file is not great, but I think we should try to improve instead of maintain.

Reply:
Should I do the documentation for the API functions of Queue Mode in this PR? Or should we all improve the documentation of lwm2m API functions in a separate PR?

Reply:
Again, up to you + team

Reply:
Lets add that soon but in a separate PR. Niclas and I should do the work on adding the doxygen on most of the APIs as we did the initial lwm2m PR.